src/views/ServiceRequest.js

```diff
@@ -32,15 +32,63 @@ const styles = {
 class ServiceRequest extends Component {
   constructor (props) {
     super(props)
+    this.singleLineFields = ['Name', 'Email', 'Phone', 'Department']
+    this.multiLineFields = [
+      'Project Description',
+      'Project Goal',
+      'Project Budget',
+      'Key Messages',
+      'Primary Target Audience',
+      'Secondary Target Audience',
+      'Project Contact (if other than yourself)',
+      'Comments'
+    ]
+    this.leftCheckboxes = [
+      'Simple1',
+      'Simple2',
+      'Simple3',
+      'Simple4',
+      'Simple5',
+      'Simple6'
+    ]
+    this.rightCheckboxes = [
+      'Simple7',
+      'Simple8',
+      'Simple9',
+      'Simple10',
+      'Simple11',
+      'Simple12'
+    ]
+    const stringProps = [
+      ...this.singleLineFields,
+      ...this.multiLineFields
+    ].reduce(
+      (acc, label) => ({
+        [this.formatLabelToProperty(label)]: '',
+        ...acc
+      }),
+      {}
+    )
+    const checkboxProps = [
+      ...this.leftCheckboxes,
+      ...this.rightCheckboxes
+    ].reduce(
+      (acc, label) => ({
+        [this.formatLabelToProperty(label)]: false,
+        ...acc
+      }),
+      {}
+    )
     this.state = {
-      fileInput: null,
-      name: '',
-      email: '',
-      phone: ''
+      fileInput: null
     }
+    Object.assign(this.state, stringProps, checkboxProps)
```

Comment:
Now that we have our objects that contain our desired properties (stringProps and checkboxProps), we can add them to the state with `Object.assign` which just adds the properties from the `2..n` arguments to the object passed as the 1st argument.